### PR TITLE
How to Build a Clock with JavaScript and SVG

### LINKED ISSUE #1686: How to Build a Clock with JavaScript and SVG
Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles to go over.

**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

*We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.*

**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

### Proposed title of the article
How to Build a Clock with JavaScript and SVG

### Introduction paragraph (2-3 paragraphs):

JavaScript is a programming language that allows you to program complex features on web pages. Scalable Vector Graphics (SVG) is an Extensible Markup Language (XML)-based vector image format for two-dimensional graphics with support for interactivity and animation. 
Using SVG (Scalar Vector Graphics) to build a clock on the web becomes easier using javascript.

### Key takeaways: 

Learn JavaScript Basics
Learn how to configure real-time on the browser using JavaScript
Readers will understand how to use SVG on the web

### References: 
https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics
https://www.w3schools.com/graphics/svg_intro.asp

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Olanetsoft**
### Article checklist
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [ ] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

fixes #1686

### CODE REVIEW

File: articles/how-to-build-a-clock-with-javascript-and-svg/index.md [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+- Conclusion.
+
+### Prerequisites
+
+- Basic understanding of JavaScript.
+- Integrated Development Environment (eg. Visual studio code).
+
+### Introduction to JavaScript
+
+JavaScript is a programming language that lets you create complex web page features. This occurs in games, in answer actions when buttons are pressed or when data is entered on forms, dynamic styling, animation, and so on.
```
Comment (WanjaMIKE) [inline articles/how-to-build-a-clock-with-javascript-and-svg/index.md:23]: Could please revise this sentence
This occurs in games, `in answer actions` when buttons are pressed or when data is entered on forms, dynamic styling, animation, and so on.

File: articles/how-to-build-a-clock-with-javascript-and-svg/index.md [hunk 2]
```diff
@@ -0,0 +1,486 @@
...
+- SVG is a free and open format.
+- SVG files are made up entirely of XML.
+
+### How to implement JavaScript with SVG on the web
+
+Now, using JavaScript and one of the SVG examples from above, let's create a simple webpage to develop a clock on the web.
+
+**Step 1:** Navigate to any Directory of your choice on your pc, in the terminal, type:
+
+```Bash
```
Comment (WanjaMIKE) [inline articles/how-to-build-a-clock-with-javascript-and-svg/index.md:218]: Inform the user that `code .` will not work if they do not have visual studio code installed

**Review by WanjaMIKE — CHANGES_REQUESTED** [review]
@Olanetsoft Hey, I have learned a lot about html and SVG. Could you please work on the suggestions then we move to the next level. Also, change the article's format like this:

### Step 1 - Creating index.html file
Content
### Step 2 - brief title
`Content

And so on.

File: articles/how-to-build-a-clock-with-javascript-and-svg/index.md (hunk 1 above)
Comment (Olanetsoft) [reply articles/how-to-build-a-clock-with-javascript-and-svg/index.md]: Done

File: articles/how-to-build-a-clock-with-javascript-and-svg/index.md (hunk 2 above)
Comment (Olanetsoft) [reply articles/how-to-build-a-clock-with-javascript-and-svg/index.md:220]: Done

**Comment by Olanetsoft** [comment]
@WanjaMIKE  Changes completed

**Review by WanjaMIKE — APPROVED** [review]
Nice article. I have learned a lot about JavaScript and SVG.

**Comment by hectorkambow** [comment]
plagiarism check done

**Pull request merged**